Bonsai/SystemResourcesExtensionsSupport.cs

```diff
@@ -13,22 +14,30 @@ namespace Bonsai
     /// </remarks>
     internal static class SystemResourcesExtensionsSupport
     {
+        // Assembly.Load(byte[]) always results in a new assembly so we need to manually de-duplicate them
+        private static ConcurrentDictionary<string, Assembly> resolveCache = new();
```

Comment:
Separately, I was wondering whether we should move `resolveCache` inside the body of the `Initialize` method? This would prevent initializing an unused concurrent dictionary in .NET Core since it would be guarded by the `ConditionalAttribute`.

Reply:
Good idea. That mildly annoyed me too but not long enough for me to think of that solution 😅